.github/workflows/deploy.yml

```diff
@@ -15,9 +15,7 @@ jobs:
       api: true
       postgres_unit_testing: true
       db_name: fsd_account_store_test
-      deploy_to_dev: false
-      users: 2
-      spawn-rate: 2
```

Comment:
Can you re-add users and spawn-rate please? @gio-karunakaran 

Some repos needs to have different locust configs for the performance tests. This will override what we have set in the workflows repo for this repo only.

Reply:
@gio-karunakaran Yes would be useful to keep them in case we need to change the config around for the account store if we need to set them differently from what's set in the workflows repo.

They are being used in run-shared-tests.yml in the workflow repo.

Reply:
separate ticket will be raised as discussed with @rayhan15243 to vary spawn rate, users and runtime in testing yml